repos/fdbt-site/tests/pages/api/pricingPerDistance.test.ts

```diff
@@ -44,14 +51,41 @@ describe('pricingPerDistance', () => {
             mockWriteHeadFn: writeHeadMock,
         });
 
-        pricingPerDistance(req, res);
+        await pricingPerDistance(req, res);
 
         expect(updateSessionAttributeSpy).toBeCalledWith(req, PRICING_PER_DISTANCE_ATTRIBUTE, mockCapInfo);
 
         expect(writeHeadMock).toBeCalledWith(302, { Location: '/additionalPricingStructures' });
     });
 
-    it('produces an error when distanceTo is empty', () => {
+    it('correctly generates cap info, updates the ticket and then redirects to /product details if all is valid', async () => {
```

Comment:
Not cap info, its pricing by distance

Reply:
removed the cap in the test name and do we need to remove it from the body also, if yes then we need to change in the page also